packages/next-sitemap/src/url/create-url-set/index.ts

```diff
@@ -97,6 +97,10 @@ export const createUrlSet = async (
 
   urlSet = [...new Set(urlSet)]
 
+  // Remove routes which don't exist
+  const notFoundRoutes = manifest.preRender?.notFoundRoutes as string[]
```

Comment:
@sreetamdas Thanks for this PR. Can you update this line with a fallback array to offer backward compatibility? (In case pervious version of `next.js` `PreRenderManifest` doesn't  include notFoundRoutes)

```js
  const notFoundRoutes = (manifest.preRender?.notFoundRoutes ?? []) as string[]
```

Reply:
Great catch @iamvishnusankar! I've added the fallback array, and a reference to when it was introduced as well :)